web/src/user_topics_ui.ts

```diff
@@ -72,15 +72,6 @@ export function handle_topic_updates(user_topic_event: ServerUserTopic): void {
             settings_user_topics.populate_list();
         }
     }
-
-    setTimeout(() => {
-        // Defer updates for any background-rendered messages lists until the visible one has been updated.
-        for (const list of message_lists.all_rendered_message_lists()) {
-            if (list.preserve_rendered_state && message_lists.current !== list) {
-                list.update_muting_and_rerender();
-            }
-        }
-    }, 0);
```

Comment:
@amanagr is removing this correct?

Reply:
I also started a conversation on CZO about this: https://chat.zulip.org/#narrow/stream/6-frontend/topic/preserve_rendered_state

Reply:
We should just remove the list.preserve_rendered_state check here. Must have missed it during the migration in [e530af50136f938ccec92333db11f9bb8b0a4459](https://github.com/zulip/zulip/commit/e530af50136f938ccec92333db11f9bb8b0a4459).

Reply:
`list.preserve_rendered_state` wouldn't have been true though, should `list.update_muting_and_rerender();` be sometimes running even though it wasn't before?

Reply:
Yeah, we should be running it.